King-Benx/laboremus-challenge
# create app icons

**Overview**
The app should be easily identifiable from the app screen

**Description**
The app should have a custom icon to identify it from the home screen.

**How to test**
1. Clone the app `git clone git@github.com:King-Benx/laboremus-challenge.git`
2. Checkout to ft-setup-app-icons `git checkout ft-setup-app-icons`
3. Install dependencies `yarn`
4. Run development server ` react-native start`
5. Run the application `react-native run-android`
6. Close the app and go to your app screen, you should be able to see an app with Sante
